app/build.gradle

```diff
@@ -61,7 +61,6 @@ dependencies {
     implementation 'androidx.compose.material:material-icons-extended'
 
     implementation 'com.squareup.okhttp3:okhttp:4.10.0'
-    implementation 'com.tinder.scarlet:scarlet:0.1.12'
```

Comment:
Should we keep this? One nice thing about that lib is that it'll parse the payloads into kotlin POJOS